traffic_ops/traffic_ops_golang/crconfig/handler.go

```diff
@@ -296,11 +303,11 @@ func SnapshotOldGUIHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	api.CreateChangeLogRawTx(api.ApiChange, "Snapshot of CRConfig performed for "+cdn, inf.User, inf.Tx.Tx)
-	http.Redirect(w, r, "/tools/flash_and_close/"+url.PathEscape("Successfully wrote the CRConfig.json!"), http.StatusFound)
+	http.Redirect(w, r, "/api/1.1/cdns/"+strconv.Itoa(cdnID)+"/snapshot", http.StatusFound)
 }
 
-func writePerlHTMLErr(w http.ResponseWriter, r *http.Request, tx *sql.Tx, logErr error, err error) {
+func writePerlHTMLErr(w http.ResponseWriter, r *http.Request, tx *sql.Tx, logErr error, err error, cdnId int) {
 	log.Errorln(logErr.Error())
 	tx.Rollback()
-	http.Redirect(w, r, "/tools/flash_and_close/"+url.PathEscape("Error: "+err.Error()), http.StatusFound)
+	http.Redirect(w, r, "/api/1.1/cdns/"+strconv.Itoa(cdnId)+"/snapshot", http.StatusFound)
```

Comment:
When there's an error that's safe for the user to see, we should show that error to the user as an alert, not redirect.